pkg/k8s/k8s.go

```diff
@@ -26,6 +26,12 @@ const (
 
 	// waitRetry is the time to wait between retries for a readiness checking
 	waitRetry = 2 * time.Second
+
+	// CustomQPS is the QPS to use for the Kubernetes client, DefaultQPS: 5
+	CustomQPS = 100
```

Comment:
We could do that as well if we are not going to change it. Otherwise we could have a config and `knuu` New configures it somehow. I think we can keep it as it is for the moment.